backend/app/services/knowledge/knowledge_service.py

```
@@ -128,69 +128,110 @@ def create_knowledge_base(
+        linked_group = data.linked_group
+        kb_name = data.name
+
+        if linked_group:
+            # Validate linked_group exists
+            from app.models.namespace import Namespace
+
+            group = (
+                db.query(Namespace)
+                .filter(
+                    Namespace.name == linked_group,
+                    Namespace.is_active == True,
```

Ruff (0.15.6)

[error] Avoid equality comparisons to `True`; use `Namespace.is_active:` for truth checks

Replace with `Namespace.is_active`

(E712)